packages/synapse-react-client/src/lib/containers/ProgrammaticInstructionsModal.tsx

```diff
@@ -53,111 +53,99 @@ export const ProgrammaticInstructionsModal = ({
+            event: React.SyntheticEvent,
+            newValue: ProgrammaticOptionsTabs,
+          ) => {
+            event.stopPropagation()
+            setCurrentTab(newValue)
+          }}
+          textColor="secondary"
+          indicatorColor="secondary"
+        >
           {cliCode && (
-            <div
-              className="Tab"
-              role="tab"
-              onClick={e => {
-                e.stopPropagation()
-                setCurrentTab(ProgrammaticOptionsTabs.COMMAND_LINE)
-              }}
-              aria-selected={
-                ProgrammaticOptionsTabs.COMMAND_LINE === currentTab
-              }
-            >
-              {ProgrammaticOptionsTabs.COMMAND_LINE}
-            </div>
+            <Tab
+              value={ProgrammaticOptionsTabs.COMMAND_LINE}
+              label={ProgrammaticOptionsTabs.COMMAND_LINE}
+            />
           )}
           {rCode && (
-            <div
-              className="Tab"
-              role="tab"
-              onClick={e => {
-                e.stopPropagation()
-                setCurrentTab(ProgrammaticOptionsTabs.R)
-              }}
-              aria-selected={ProgrammaticOptionsTabs.R === currentTab}
-            >
-              {ProgrammaticOptionsTabs.R}
-            </div>
+            <Tab
+              value={ProgrammaticOptionsTabs.R}
+              label={ProgrammaticOptionsTabs.R}
+            />
           )}
           {pythonCode && (
-            <div
-              className="Tab"
-              role="tab"
-              onClick={e => {
-                e.stopPropagation()
-                setCurrentTab(ProgrammaticOptionsTabs.PYTHON)
-              }}
-              aria-selected={ProgrammaticOptionsTabs.PYTHON === currentTab}
-            >
-              {ProgrammaticOptionsTabs.PYTHON}
-            </div>
-          )}
-        </div>
-        <div className="TabContent">
-          {currentTab === ProgrammaticOptionsTabs.COMMAND_LINE && (
-            <>
-              <p>{cliNotes}</p>
-              {installationInstructions}
-              <pre> {cliCode} </pre>
-            </>
-          )}
-          {currentTab === ProgrammaticOptionsTabs.R && (
-            <>
-              <p>{rNotes}</p>
-              {installationInstructions}
-              <pre>{rCode}</pre>
-            </>
+            <Tab
+              value={ProgrammaticOptionsTabs.PYTHON}
+              label={ProgrammaticOptionsTabs.PYTHON}
+            />
           )}
-          {currentTab === ProgrammaticOptionsTabs.PYTHON && (
-            <>
-              <p>{pythonNotes}</p>
-              {installationInstructions}
-              <pre>{pythonCode}</pre>
-            </>
-          )}
-        </div>
-      </Modal.Body>
-      <Modal.Footer>
-        <Button variant="primary" onClick={onClose}>
-          OK
-        </Button>
-      </Modal.Footer>
-    </Modal>
+        </Tabs>
+      </Box>
+      <Box sx={{ '& > p': { marginBottom: 3 } }}>
```

Comment:
Apply margin to paragraphs, since no longer provided by bootstrap-4-backport -- 

<img width="1060" alt="feature_ProgrammaticInstructionsModal" src="https://user-images.githubusercontent.com/26949006/234135316-12a9e5cd-357b-4cec-836f-522a2f209b1e.png">